projects/turso.tech/package.yml

```diff
@@ -0,0 +1,37 @@
+distributable:
+  url: https://github.com/tursodatabase/homebrew-tap/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: tursodatabase/homebrew-tap
+
+provides:
+  - bin/turso
+
+build:
+  script: |
+    git clone https://github.com/tursodatabase/turso-cli
```

Comment:
Instead of this, either you should make the `distributable.url: https://github.com/tursodatabase/turso-cli/archive/refs/tags/v{{version}}.tar.gz` (and use the same github for the version), or make the `distributable: { url: git+https://github.com/tursodatabase/turso-cli, ref: v{{version}} }`, if it _strictly requires_ being built from a git checkout (rare, but it exists).

Reply:
I think I tried the former but ended up getting errors around version parsing

```
no versions parsed. Re-run with DEBUG=1 to see output.
{ versions: [] }
error: Uncaught Error: not-found: version: turso.tech
```

Reply:
Looks like setting the version depends on passing `-tags=prod` to the build command.